# 客户端请求时发送的两个数字
bool vision_flag  # 请求中包含一个布尔值，表示视觉标志位
bool fcu_flag  # 请求中包含一个布尔值，表示飞控标志位
---
# 服务器响应发送的数据
bool success       # 响应中包含一个布尔值，表示操作是否成功
bool vision_flag  # 响应中包含一个布尔值，表示视觉标志位
bool fcu_flag  # 响应中包含一个布尔值，表示飞控标志位